### DISCUSSION TITLE
blog/843fc0c8/thoughts-on-rebuilding-my-personal-website

### DISCUSSION THREAD

**Discussion by giscus** [laniakita/website#8] [Announcements]
# blog/843fc0c8/thoughts-on-rebuilding-my-personal-website

This article was a long time coming, it's also quite long. In some respects, this might feel like an article created from a few separate articles I stitched together, and perhaps it is. However, I felt it was important to talk about the primary technologies I used in developing/deploying this site, because they're all pretty tightly integrated.

https://laniakita.com/blog/843fc0c8/thoughts-on-rebuilding-my-personal-website

**Comment by laniakita** [comment]
Commenting now works! If you have a comment, question, or concern, please don't hesitate to share your thoughts!
